lib/cli/test/commands/init.test.js

```diff
@@ -0,0 +1,49 @@
+const fs = require('fs');
+const path = require('path');
+const rimraf = require('rimraf');
+
+const { run, copyDirSync } = require('../helpers');
+
+const fixturesDirPath = path.join(__dirname, '..', 'fixtures');
+const runDirPath = path.join(__dirname, '..', 'run');
+
+const rootPath = path.join(__dirname, '..', '..', '..', '..');
+
+beforeAll(() => {
+  fs.mkdirSync(runDirPath);
+  // Copy all files from fixtures directory to `run`
+  const dirs = fs.readdirSync(fixturesDirPath);
+  dirs.forEach((dir) => copyDirSync(path.join(fixturesDirPath, dir), runDirPath));
+});
+
+afterAll(() => {
+  rimraf.sync(runDirPath);
```

Comment:
We should find a way to clean things if users exit command in the middle of the tests. Or maybe just clean the directory in the beforeAll if it exists?